hotsos.sh

```diff
@@ -140,10 +142,14 @@ OPTIONS
     --user-summary
         Used in conjunction with --short to allow an existing (full) summary to
         to be shortened.
-
+    --format [yaml|json]
+        The format of the output. Defaults to yaml.
+    --yaml
+        Output in yaml format.
+    --json
+        Output in json format.
```

Comment:
Lets just use --json, the rest are all redundant

Reply:
In a way it is redundant since `yaml` is the default, that's true. But shouldn't the user be able to specify the default output format even if it's the default and that choice is redundant? This is a common pattern, see e.g. the openstack or the juju clients.

Reply:
Ok will remove.